manifests/config/sharedport.pp

```diff
@@ -0,0 +1,22 @@
+# Configuration of Condor SharedPort service
+class htcondor::config::sharedport {
+  $use_shared_port            = $htcondor::use_shared_port
+  $shared_port                = $htcondor::shared_port
+  $shared_port_collector_name = $htcondor::shared_port_collector_name
+
+  $template_sharedport        = $htcondor::template_sharedport
+  $managers                   = $htcondor::managers
+
+  $now                        = strftime('%d.%m.%Y_%H.%M')
+
+  # SharedPort service configuration
+  file { '/etc/condor/config.d/42_shared_port.config':
+    backup  => ".bak.${now}",
+    content => template($template_sharedport),
+    require => Package['condor'],
+    owner   => $condor_user,
+    group   => $condor_group,
```

Comment:
Puppet lint is complaining about lines 17 and 18:
```
manifests/config/sharedport.pp - WARNING: top-scope variable being used without an explicit namespace on line 17
manifests/config/sharedport.pp - WARNING: top-scope variable being used without an explicit namespace on line 18
```

Could you please add 
```
$condor_user                = $htcondor::condor_user
$condor_group               = $htcondor::condor_group
```
?